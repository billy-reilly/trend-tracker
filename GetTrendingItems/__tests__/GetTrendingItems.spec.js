describe("tests", () => {
  it("should pass", () => {
    expect(true).toBe(true);
  });

  it("should fail", () => {
    expect(true).toBe(false);
  });
});
