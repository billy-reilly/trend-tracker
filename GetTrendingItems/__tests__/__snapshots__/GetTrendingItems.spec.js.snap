// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetTrendingItems should query DynamoDB 1`] = `
Object {
  "ConsistentRead": true,
  "ExpressionAttributeValues": Object {
    ":tl": Object {
      "S": "shoes",
    },
  },
  "IndexName": "trendListId-interactionCount-index",
  "KeyConditionExpression": "trendListId = :tl",
  "Limit": "3",
  "ProjectionExpression": "itemId, interactionCount",
  "ScanIndexForward": false,
  "TableName": "InteractionCounts",
}
`;
